src/components/Picker/Picker.scss

```diff
@@ -149,4 +149,19 @@ $stacking-order: (
     line-height: 30px;
   }
 
+}
+
+.error {
+  
+  .input::placeholder {
+    // This is the only place this color is used.
+    // stylelint-disable-next-line color-no-hex
+    color: #9c9798;
+  }
+
+  .backdrop {
+    background-color: color(error, light);
```

Comment:
Add error styling in same manor as for other components. Ex: https://github.com/emgage/engage-ui/blob/master/src/components/TextField/TextField.scss#L59

Make certain to also include Delicious theme styling. Ex: https://github.com/emgage/engage-ui/blob/master/themes/Delicious/TextField.scss#L60

Reply:
@weegrom the changes is done